lib/ansiblelint/__init__.py

```diff
@@ -66,7 +64,7 @@ def matchlines(self, file, text):
             if not result:
                 continue
             message = None
-            if isinstance(result, six.string_types):
+            if isinstance(result, str):
```

Comment:
Let's revert all the modifications to `six.string_types` for now as what you're doing effectively modifies logic. Before all checks were "bytes or unicode" not it's just "unicode" which may lead to unforeseen behavior changes. So I want to see less of dangerous changes in this PR.

Reply:
I really doubt there is any change in behavior, take a look at https://six.readthedocs.io/#six.string_types -- I did this switch on other projects.

As long we run on python3 there is no "unicode" anymore. The only case where it would be a change in behavior is if we endup running on py2, which we do not allow or support.

Reply:
Okay, keep it then.